parachain/pallets/commitments/src/lib.rs

```diff
@@ -105,11 +105,11 @@ impl<T: Config> Module<T> {
 		let commitment = Self::encode_commitment(&messages);
 		let commitment_hash: H256 = keccak_256(&commitment).into();
 
-		<Self as Store>::Commitment::set(commitment);
-
 		let digest_item = CustomDigestItem::Commitment(commitment_hash.clone()).into();
 		<frame_system::Module<T>>::deposit_log(digest_item);
 
+		sp_io::offchain_index::set(T::INDEXING_KEY, &commitment);
```

Comment:
is this setting a new key for each commitment, or overwriting the same indexing key? we should be indexing by the commitment_hash so that we can query with the commitment_hash as input, right?

Reply:
Good catch, that is the better approach. Will update